### ISSUE TITLE
zuul --help does not print the full usage guide on node v6

### ISSUE THREAD

**Issue by 75lb** [defunctzombie/zuul#282]
this is the node 6 output, note it is incomplete:
```
$ zuul -h

  Usage: zuul [options] <files | dir>

  Options:

    -h, --help                             output usage information
    -V, --version                          output the version number
    --ui <testing ui>                      ui for tests (mocha-bdd, mocha-tdd, qunit, tape)
    --local [port]                         port for manual testing in a local browser
    --tunnel [type]                        establish a tunnel for outside access. only used when --local is specified
    --disable-tunnel                       don't establish a tunnel for outside access. override any config in .zuul.yml and .zuulrc
    --phantom [port]                       run tests in phantomjs. PhantomJS must be installed separately.
    --phantom-remote-debugger-port [port]  connect phantom to remote debugger
    --phantom-remote-debugger-autorun      run tests automatically when --phantom-remote-debugger-port is specified
    --electron                             run tests in electron. electron must be installeecc-ui
```

this is related to [updated process.exit() semantics](https://nodejs.org/dist/latest-v6.x/docs/api/process.html#process_process_exit_code) in node v6.. essentially, you need to let the node process exit gracefully - `process.exit()` now kills the process on the spot without flushing `process.stdout`

**Comment by vvo** [comment]
Do you think you can submit a PR to solve this?

**Comment by lpinca** [comment]
Should be fixed upstream, see https://github.com/tj/commander.js/issues/530.

**Comment by lpinca** [comment]
I think this can be closed. It should have been fixed in node 6.2.1
